terraform/test/auth_preference_test.go

```diff
@@ -92,3 +104,38 @@ func (s *TerraformSuite) TestImportAuthPreference() {
 		},
 	})
 }
+
+func (s *TerraformSuiteWithCache) TestAuthPreferenceAddLabel() {
+	name := "teleport_auth_preference.cluster_auth_preference"
+
+	resource.Test(s.T(), resource.TestCase{
+		ProtoV6ProviderFactories:  s.terraformProviders,
+		PreventPostDestroyRefresh: true,
+		Steps: []resource.TestStep{
+			{
+				Config: s.getFixture("auth_preference_0_cluster.tf"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(name, "kind", "cluster_auth_preference"),
+					resource.TestCheckNoResourceAttr(name, "metadata.labels"),
+					resource.TestCheckResourceAttr(name, "spec.type", "oidc"),
+				),
+			},
+			// {
```

Comment:
What's up with these commented-out bits?

Reply:
Totally forgot to stage that change into the git commit
Sorry